### ISSUE TITLE
Enchantments doing nothing

### ISSUE THREAD

**Issue by deathzero021** [HxCKDMS/HxCEnchants#49]
I have HxC-Core and HxC-Enchantments and yet all of the enchantments seem to do nothing in Minecraft 1.7.10. I downloaded and installed the mods using Curse for Minecraft. I have 32 mods total running and all other mods run fine, no bugs or crashes. Also i can't seem to find anything information on what the "charge system" is or what it does.

**Comment by DrZed** [comment]
"Charge" is required for any enchant to work, just sneak while holding an enchanted item in hand to convert xp to charge. Each enchant has a configurable amount of charge required.
You can either A.) Change requirements to 0 B.) Disable charge system
I use this mod on my server and my enchants do work.

**Comment by deathzero021** [comment]
Okay so I disabled the charge system in the config file and after restarting the game, enchanted items no longer show the "charges: x" in the tool tip but when I press Sneak it still drains my levels and the new enchantments still don't do anything at all. I may try testing the mod separately in a new world to see if this problem is a conflict with another mod in my current world.

Edit: I just tried running HxC-core and HxC-Enchantments by in it's own profile and Minecraft crashes when loading the mods. Crash report:

/////////////////////////////////////////////////////////////////////////////////////////////////////////////
---- Minecraft Crash Report ----
// This is a token for 1 free hug. Redeem at your nearest Mojangsta: [~~HUG~~]

Time: 8/16/16 9:04 AM
Description: There was a severe problem during mod loading that has caused the game to fail

cpw.mods.fml.common.LoaderException: java.lang.NoSuchMethodError: HxCKDMS.HxCCore.api.Configuration.HxCConfig: method <init>()V not found
    at cpw.mods.fml.common.LoadController.transition(LoadController.java:163)
    at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:559)
    at cpw.mods.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:243)
    at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:480)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:878)
    at net.minecraft.client.main.Main.main(SourceFile:148)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.NoSuchMethodError: HxCKDMS.HxCCore.api.Configuration.HxCConfig: method <init>()V not found
    at HxCKDMS.HxCBlocks.HxCBlocks.preInit(HxCBlocks.java:29)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at cpw.mods.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:532)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at cpw.mods.fml.common.LoadController.sendEventToModContainer(LoadController.java:212)
    at cpw.mods.fml.common.LoadController.propogateStateMessage(LoadController.java:190)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:483)
    at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
    at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
    at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
    at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
    at com.google.common.eventbus.EventBus.post(EventBus.java:275)
    at cpw.mods.fml.common.LoadController.distributeStateMessage(LoadController.java:119)
    at cpw.mods.fml.common.Loader.preinitializeMods(Loader.java:556)
    ... 10 more

## A detailed walkthrough of the error, its code path and all known details is as follows:

-- System Details --
Details:
    Minecraft Version: 1.7.10
    Operating System: Windows 10 (amd64) version 10.0
    Java Version: 1.8.0_25, Oracle Corporation
    Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
    Memory: 691725056 bytes (659 MB) / 900726784 bytes (859 MB) up to 3817865216 bytes (3641 MB)
    JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx4096m -Xms256m -XX:PermSize=256m
    AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
    IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
    FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1614 7 mods loaded, 7 mods active
    States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
    UCH mcp{9.05} [Minecraft Coder Pack](minecraft.jar) 
    UCH FML{7.10.99.99} [Forge Mod Loader](forge-1.7.10-10.13.4.1614-1.7.10.jar) 
    UCH Forge{10.13.4.1614} [Minecraft Forge](forge-1.7.10-10.13.4.1614-1.7.10.jar) 
    UCH HxCCore-ASM{1.12.4} [HxCKDMS Core ASM](minecraft.jar) 
    UCH HxCCore{1.12.4} [HxC-Core](HxCCore-1.7.10-1.12.4-universal.jar) 
    UCE HxCBlocks{1.6.2} [HxC-Blocks](HxC-Blocks-1.7.10-1.6.2.jar) 
    UCH HxCEnchants{2.3.4} [HxC Enchants](HxCEnchants-1.7.10-2.3.4.jar) 
    GL info: ' Vendor: 'ATI Technologies Inc.' Version: '4.5.13447 Compatibility Profile Context 20.19.0.32832' Renderer: 'AMD Radeon (TM) R7 360 Series'
    HxCKDMS Crash Check: Will analyze crash log and send the error to github if HxCKDMS Core is possibly involved.

**Comment by DrZed** [comment]
HxCBlocks is the reason it's crashing, I haven't finished the rewrite of the mod, so it's currently broken.

**Comment by DrZed** [comment]
You don't have to send me the crash reports you know, usually if it's my fault I get the crash reports in 
https://github.com/HxCKDMS/AutomaticCrashReports/issues

**Comment by deathzero021** [comment]
okay so i removed hxc-blocks and it runs now. testing the enchantments via creative mode in a fresh world and it seems like the majority of them do nothing at all. Vampirism did seem to do a little bit of hunger healing and regeneration did heal me very slowly when my hunger was low. Other than that, the other enchantments didn't work. I tested quite a few of them like Jump Boost, did not improve my jump. Homing did not cause arrows to follow or even curve towards targets. Nightvision failed, etc. I'm thinking maybe each enchantment has to be on a specific item to work but i have no idea which enchantments go where. I tested them on a few different items, from weapons, bows and armors. The Aura's i put on armor and did nothing. Combat based ones i put on weapons. Buffs i put on armor as well. I charged all of the enchanted items to over 10k just to be sure there was enough charge. So i don't have a clue why it isn't working. 

In my main world, i have removed this mod because the charge system was still draining my lvls even though i disabled it. So for now, i'm done trying this one out. All testing I do shows negative results, aside from Regen and Vampirism.
